src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -8,18 +8,23 @@
       <button *ngIf="rundown?.isActive" (click)="deactivateRundown()">Deactivate Rundown</button>
     </div>
     <div class="segments">
-      <sofie-segment *ngFor="let segment of rundown?.segments" [segment]="segment" [isRundownActive]="rundown?.isActive ?? false" (setNextEvent)="setNext($event)"></sofie-segment>
-    </div>
-  </div>
-  <div>
-    <div class="infinite-pieces">
-      <span class=subtitle>Infinite Pieces</span>
-      <sofie-piece *ngFor="let infinitePiece of rundown?.getInfinitePieces()" [piece]="infinitePiece"></sofie-piece>
-    </div>
-    <div class="ad-lib-pieces">
-      <span class=subtitle>Adlib Pieces</span>
-      <sofie-ad-lib-piece-identifier *ngFor="let identifier of adLibPieceIdentifiers" [adLibPieceIdentifier]="identifier" [rundownId]="rundown?.id ?? ''"></sofie-ad-lib-piece-identifier>
+      <sofie-segment
+        *ngFor="let segment of rundown?.segments"
+        [segment]="segment"
+        [isOnAir]="segment.isOnAir"
+        (setNextEvent)="setNext($event)"
+      ></sofie-segment>
     </div>
   </div>
+<!--  <div>-->
+<!--    <div class="infinite-pieces">-->
+<!--      <span class=subtitle>Infinite Pieces</span>-->
+<!--      <sofie-piece *ngFor="let infinitePiece of rundown?.getInfinitePieces()" [piece]="infinitePiece"></sofie-piece>-->
+<!--    </div>-->
+<!--    <div class="ad-lib-pieces">-->
+<!--      <span class=subtitle>Adlib Pieces</span>-->
+<!--      <sofie-ad-lib-piece-identifier *ngFor="let identifier of adLibPieceIdentifiers" [adLibPieceIdentifier]="identifier" [rundownId]="rundown?.id ?? ''"></sofie-ad-lib-piece-identifier>-->
+<!--    </div>-->
+<!--  </div>-->
```

Comment:
Is this going to be used in the future, or just leftover code?
If suppose to be used later, then maybe add a comment detailing that?

Reply:
Great to see it gone.